tools/Libplanet.Explorer.Cocona/Libplanet.Explorer.Cocona.csproj

```diff
@@ -8,7 +8,7 @@
   <ItemGroup>
     <PackageReference Include="Cocona.Lite" Version="2.0.*" />
     <PackageReference Include="Serilog.Sinks.Console" Version="4.1.0" />
-    <PackageReference Include="System.Text.Json" Version="7.0.0" />
+    <PackageReference Include="System.Text.Json" Version="9.0.*" />
```

Comment:
Bumping to the latest version is always welcome. At least for me. 😀

Reply:
Vulnerability in 7.0 causes failure in build 😢